src/components/Section/shared/SectionNavigation.jsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+
+import { i18n, env } from '@config'
+import * as formTypeConfig from '@config/formTypes'
+
+import * as sections from '@constants/sections'
+
+import SectionNavButton from './SectionNavButton'
+
+const SectionNavigation = ({ section, subsection, formType }) => {
+  const formSections = formTypeConfig[formType]
+
+  const sectionConfig = formSections.find(s => s.key === section)
+  const sectionIndex = formSections.findIndex(s => s.key === section)
+  const subsectionIndex = sectionConfig.subsections.findIndex(s => s.name === subsection)
+
+  let back
+  let next
+  let backLabel
+  let nextLabel
+
+  if (subsectionIndex > 0) {
+    back = sectionConfig.subsections[subsectionIndex - 1]
+    backLabel = i18n.t(`${sectionConfig.name}.destination.${back.name}`)
+  } else if (sectionIndex > 0) {
+    // Go to the last subsection of the previous section
+    const previousSection = formSections[sectionIndex - 1]
+    back = previousSection.subsections[previousSection.subsections.length - 1]
+    backLabel = i18n.t(`${previousSection.name}.destination.${back.name}`)
+  }
+
+  if (subsectionIndex < sectionConfig.subsections.length - 1) {
+    next = sectionConfig.subsections[subsectionIndex + 1]
+    nextLabel = i18n.t(`${sectionConfig.name}.destination.${next.name}`)
+  } else if (sectionIndex < formSections.length - 1) {
+    // Go to the first subsection of the next section
+    const nextSection = formSections[sectionIndex + 1]
+    next = nextSection.subsections[0]
+    nextLabel = i18n.t(`${nextSection.name}.destination.${next.name}`)
+  }
+
+  if (!back && !next) {
+    return null
+  }
+
+  const backButton = back ? (
+    <SectionNavButton
+      direction="back"
+      label={backLabel}
+      onClick={() => { goToSection(`/form${back.path}`)}} />
+  ) : <SectionNavButton direction="back" isEmpty={true} />
```

Comment:
For both the back and next button, can we remove this ternary from the component level and put it in the prop level?

```
isEmpty={back ? true : false}

// or simply

isEmpty={!!back}
```
...then put the component directly in the render instead of storing it in a variable